roles/create-vms/templates/vm-template.xml.j2

```diff
@@ -3,7 +3,7 @@
   <memory unit='MB'>{{ item.memory }}</memory>
   <vcpu>{{ item.cpus }}</vcpu>  
   <os>
-    <type arch="x86_64" machine="pc-i440fx-4.1">hvm</type>
```

Comment:
why is this being removed?

Reply:
QEMU docs recommend not specifying machine, plus it just didn't work on my RHEL8 CSB laptop build with an Intel i7.